style.css

```diff
@@ -61,6 +61,17 @@ body {
   transition: 500ms ease-in;
 }
 
```

Comment:
- **Kindly fix the style of your project isn't centered.**
![Screen Shot 2021-08-04 at 17 17 58](https://user-images.githubusercontent.com/22258274/128262691-5f06bcda-1a8a-490f-91cd-1766231bd221.png)

- Remove the exceeded  "X".
![Screen Shot 2021-08-04 at 17 20 04](https://user-images.githubusercontent.com/22258274/128262763-2b4ebe77-4b30-40f3-8d10-9f01272865d4.png)